pypeit/core/wavecal/spectrographs/templ_keck_deimos.py

```diff
@@ -6,31 +6,53 @@
 
 # Keck/DEIMOS
 
-def keck_deimos_600ZD():
+def keck_deimos_600ZD(overwrite=False):
     binspec = 1
-    slits = [0, 1]
-    lcut = [7192.]
-    xidl_file = os.path.join(templates.template_path, 'Keck_DEIMOS', '600ZD', 'deimos_600.sav')
-    outroot = 'keck_deimos_600.fits'
-    templates.build_template(xidl_file, slits, lcut, binspec, outroot, lowredux=True)
+    outroot = 'keck_deimos_600ZD.fits'
+    # PypeIt fits
+    wpath = os.path.join(templates.template_path, 'Keck_DEIMOS', '600ZD')
+
+    basefiles = ['MasterWaveCalib_A_1_02_useS0896.fits', 'MasterWaveCalib_A_1_02_useS0477.fits',
```

Comment:
Have you added these to the DevSuite?  If not, please do.

Reply:
Yes, I did. I actually have a PR for that. https://github.com/pypeit/PypeIt-development-suite/pull/139